packages/react/src/components/SplitButton/SplitButton.tsx

```diff
@@ -84,6 +84,9 @@ export interface SplitButtonProps
   /** A split button can be formatted to show different levels of emphasis. */
   secondary?: boolean
 
+  /** A split button can be small */
+  small?: boolean
```

Comment:
add this to propTypes as well.

Reply:
done

Reply:
Seems like this might be better as an optional `size` prop. Right now we only need to allow for small and default sizes (and default without specifying), but it would allow flexibility to have more sizes for splitbutton in the future w/o adding more props